### PR TITLE
[ISSUE #395] Simplify notifiy Webhook

### PR DESCRIPTION
**Pull request by KamToHung**
1. We can copy robot webhook in webhook config
2. Custom webhook should url, not a url with query param(like http://xxx?key=)
3. webhook instead of webHook

### CODE REVIEW

File: common/src/main/java/org/dromara/dynamictp/common/notifier/DingNotifier.java [hunk 1]
```diff
@@ -86,11 +87,16 @@ protected String buildUrl(NotifyPlatform platform) {
      * @return url
      */
     private String getTargetUrl(String secret, String accessToken, String webHook) {
```
Comment (kazetyan) [inline common/src/main/java/org/dromara/dynamictp/common/notifier/DingNotifier.java:89]: webhook保持拼写风格一致

File: common/src/main/java/org/dromara/dynamictp/common/notifier/DingNotifier.java [hunk 2]
```diff
@@ -73,7 +74,7 @@ protected String buildMsgBody(NotifyPlatform platform, String content) {
 
     @Override
     protected String buildUrl(NotifyPlatform platform) {
-        String webHook = Optional.ofNullable(platform.getWebHook()).orElse(DingNotifyConst.DING_WEBHOOK);
+        String webHook = Optional.ofNullable(platform.getWebhook()).orElse(DingNotifyConst.DING_WEBHOOK);
```
Comment (kazetyan) [inline common/src/main/java/org/dromara/dynamictp/common/notifier/DingNotifier.java:77]: webhook保持拼写风格一致

Comment (KamToHung) [reply common/src/main/java/org/dromara/dynamictp/common/notifier/DingNotifier.java]: PTAL

**Review by kazetyan — COMMENTED** [review]
greate

**Pull request merged**